include/mbedtls/pk.h

```diff
@@ -1213,12 +1213,17 @@ int mbedtls_pk_write_pubkey(unsigned char **p, unsigned char *start,
                             const mbedtls_pk_context *key);
 #endif /* MBEDTLS_PK_WRITE_C */
 
-#if defined(MBEDTLS_USE_PSA_CRYPTO)
+#if defined(MBEDTLS_USE_PSA_CRYPTO) && !defined(MBEDTLS_DEPRECATED_REMOVED)
```

Comment:
Actually now that we've removed all uses of this function from tests and programs, I'm inclined to remove the function entirely: it was documented as "temporary, migth be removed at any time without notice", so let's take advantage of that. (With a ChangeLog entry pointing people to `get_psa_attributes()` etc.)

(If we keep it, we should test it. Doesn't seem worth the effort to me.)